Formula/opencv@2.rb

```diff
@@ -0,0 +1,87 @@
+class OpencvAT2 < Formula
+  desc "Open source computer vision library"
+  homepage "http://opencv.org/"
+  url "https://github.com/opencv/opencv/archive/2.4.13.2.tar.gz"
+  sha256 "4b00c110e6c54943cbbb7cf0d35c5bc148133ab2095ee4aaa0ac0a4f67c58080"
+  revision 2
+
+  option "without-python", "Build without python2 support"
+
+  depends_on "cmake" => :build
+  depends_on "pkg-config" => :build
+  depends_on "eigen"
+  depends_on "ffmpeg"
+  depends_on "jpeg"
+  depends_on "libpng"
+  depends_on "libtiff"
+  depends_on "openexr"
+  depends_on :python => :recommended if MacOS.version <= :snow_leopard
+  depends_on "numpy" if build.with? "python"
+
+  def install
+    jpeg = Formula["jpeg"]
+
+    args = std_cmake_args + %W[
+      -DCMAKE_OSX_DEPLOYMENT_TARGET=
+      -DBUILD_JASPER=OFF
+      -DBUILD_JPEG=OFF
+      -DBUILD_OPENEXR=OFF
+      -DBUILD_PERF_TESTS=OFF
+      -DBUILD_PNG=OFF
+      -DBUILD_TESTS=OFF
+      -DBUILD_TIFF=OFF
+      -DBUILD_ZLIB=OFF
+      -DBUILD_opencv_java=OFF
+      -DWITH_CUDA=OFF
+      -DWITH_EIGEN=ON
+      -DWITH_FFMPEG=ON
+      -DWITH_GSTREAMER=OFF
+      -DWITH_JASPER=OFF
+      -DWITH_OPENEXR=ON
+      -DWITH_OPENGL=ON
+      -DWITH_TBB=OFF
+      -DJPEG_INCLUDE_DIR=#{jpeg.opt_include}
+      -DJPEG_LIBRARY=#{jpeg.opt_lib}/libjpeg.dylib
+    ]
+
+    args << "-DBUILD_opencv_python=" + (build.with?("python") ? "ON" : "OFF")
+
+    if build.with? "python"
+      py_prefix = `python-config --prefix`.chomp
+      py_lib = "#{py_prefix}/lib"
+      args << "-DPYTHON_LIBRARY=#{py_lib}/libpython2.7.dylib"
+      args << "-DPYTHON_INCLUDE_DIR=#{py_prefix}/include/python2.7"
+
+      # Make sure find_program locates system Python
+      # https://github.com/Homebrew/homebrew-science/issues/2302
+      args << "-DCMAKE_PREFIX_PATH=#{py_prefix}"
+    end
+
+    if ENV.compiler == :clang && !build.bottle?
+      args << "-DENABLE_SSSE3=ON" if Hardware::CPU.ssse3?
+      args << "-DENABLE_SSE41=ON" if Hardware::CPU.sse4?
+      args << "-DENABLE_SSE42=ON" if Hardware::CPU.sse4_2?
+      args << "-DENABLE_AVX=ON" if Hardware::CPU.avx?
+    end
+
+    mkdir "build" do
+      system "cmake", "..", *args
+      system "make"
+      system "make", "install"
+    end
+  end
+
+  test do
+    (testpath/"test.cpp").write <<-EOS.undent
+      #include <opencv/cv.h>
+      #include <iostream>
+      int main() {
+        std::cout << CV_VERSION << std::endl;
+        return 0;
+      }
+    EOS
+    system ENV.cxx, "test.cpp", "-I#{include}", "-L#{lib}", "-o", "test"
+    assert_equal version.to_s, shell_output("./test").strip
+    assert_match version.to_s, shell_output("python -c 'import cv2; print(cv2.__version__)'")
```

Comment:
Can the openc@2 test block be also run for both Python versions with `Language::Python.each_python(build) do |python, _version|`?